config/initializers/devise.rb

```diff
@@ -5,14 +5,6 @@
   # random tokens. Changing this key will render invalid all existing
   # confirmation, reset password and unlock tokens in the database.
   config.secret_key = ENV['DEVISE_SECRET_KEY']
-  # ==> Mailer Configuration
-  # Configure the e-mail address which will be shown in Devise::Mailer,
-  # note that it will be overwritten if you use your own mailer class
-  # with default "from" parameter.
-  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```

Comment:
We don't send or receive any email from Devise, because passwords aren't recoverable over email.

Reply:
What do you think about `send_unlock_instructions` in https://github.com/plataformatec/devise/blob/7b3081760f32e7150601776276bd809601b8eaaa/lib/devise/models/lockable.rb#L42?

This would have the additional benefit of notifying a user of failed attempts on their account if it wasn't them.

Reply:
(not that they could unlock over email)